lib/cloudflare-routing.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-undef, prefer-template, no-useless-escape */
+exports.determineRouting = url => {
+  const { protocol: reqProtocol, host: reqDomain, pathname: reqPath } = new URL(url);
+
+  return INSTALL_OPTIONS.ROUTES.map(r => {
```

Comment:
Did you have any luck using minimatch for this? Are we okay with the greatly simplified code if it means that one case in the docs won't be handled?

Reply:
Everything worked besides the last test case in `test/cloudflare-routing.test.js`. I think it's your call.

Reply:
Just for future me, these are the cases that would all work correctly with cloudflare's routing:

```js
const minimatch = require('minimatch')
> minimatch('https://example.com/path', 'https://example.com/path*')
true
> minimatch('https://example.com/path2', 'https://example.com/path*')
true
> minimatch('https://example.com/path/readme.txt', 'https://example.com/path*')
false
```

But the last one fails. If you really did want to do this, a workaround would be to just do this as well:

```
https://example.com/path/*
```

Which then works:

```js
> minimatch('https://example.com/path/readme.txt', 'https://example.com/path/*')
true
```

Since there's a workaround, i'm okay with this limitation as it makes our code much simpler. Lets migrate this to using minimatch! Examples taken from cloudflare docs: https://workers.cloudflare.com/docs/reference/workers-concepts/routes/

Reply:
Fair enough. It won't be a 1-for-1 but it is cleaner! We can document this for our users.

Reply:
👍 I think realistically people will just have like `https://example.com/api/*`anyway.